include/mcmini/misc/cond/MCConditionVariablePolicy.hpp

```diff
@@ -116,6 +116,8 @@ class ConditionVariablePolicy {
    */
   virtual void add_waiter(tid_t tid) = 0;
 
+  virtual bool has_waiters() = 0;
```

Comment:
Presumably `has_waiters()` won't modify the policy, so this should be marked a `const` member function as so
```cpp
virtual bool has_waiters() const = 0;
```